lib/plugin/videoroom/errors.ex

```diff
@@ -0,0 +1,22 @@
+defmodule Janus.Plugin.VideoRoom.Errors do
```

Comment:
I would imagine this module being a handler that is common for all the plugin calls. All errors are standardized. One could just make translation from `janus error` to our `{:error, reason}`

Reply:
Is it necessary when we use `handle_videoroom_error` inside videoroom module? Or just move this function inside `Errors` module?

Reply:
It's up to you to decide. Moving it in is ok. Although a function like this will be used anywhere, so if we could extract logic it could save time in the long run.